static/js/api/instances.tsx

```diff
@@ -160,3 +160,35 @@ export const fetchInstanceVga = (
       .catch(reject);
   });
 };
+
+export const fetchInstanceConsoleBuffer = (
+  name: string,
+  project: string
+): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    fetch(`/1.0/instances/${name}/console?project=${project}`, {
+      method: "GET",
+    })
+      .then(handleTextResponse)
+      .then((data: string) => resolve(data))
+      .catch(reject);
+  });
+};
+
+export const fetchInstanceConsole = (
```

Comment:
It's a bit weird to have this called "fetch"; shouldn't it use the word "connect" instead? (Like [in the API specs](https://linuxcontainers.org/lxd/docs/latest/api/#/instances/instance_console_post))